tensornetwork/network.py

```diff
@@ -16,20 +16,21 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
-import collections
 import h5py
 # pylint: disable=line-too-long
 from typing import Any, Sequence, List, Set, Optional, Union, Text, Tuple, Type, Dict, BinaryIO
 import numpy as np
-import weakref
 from tensornetwork import config
-from tensornetwork import network_components
+#pylint: disable=useless-import-alias
```

Comment:
Why did you disable this check?

Reply:
how can I make network_components available?